.github/workflows/ci_test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: [3.7, 3.8, 3.9, '3.10', `3.11`]
```

Comment:
I think this needs to be single quotes not backticks.

Reply:
Good point!  Turns out those keys are actually very close to each other in Dvorak.